usagov/usagov-2021
# USAGOV-1917-apply-selections-active:

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1917

## Description

Set active color to same as hover for apply selection button
## Type of Changes


![image](https://github.com/user-attachments/assets/8c346a72-b24c-4dd8-bcbb-f5d52cb70cd8)


## Testing Instructions



### Change Requirements



### Validation Steps

- Go to benefit search page, click on "Apply Selecttions" button and confirm the background color is same as when hovering. It should not change to white when you press the button.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
